docs/Installation.md

```diff
@@ -41,34 +40,55 @@ minutes to complete.
 Building and flashing the micro-controller
 ==========================================
 
+Via Octoprint, make sure that there is no active serial connection to the printer.
+Fom the OctoPrint web page, under the "Connection" section, click "Disconnect"
+
```

Comment:
This is not yet necessary at this stage and has already been addressed later in the document, at the appropriate time.

Reply:
What would make it necessary later? The only reason you would check is if you were already using Octoprint with the printer before deciding to change to Klipper. Octoprint won't auto-connect  to the printer during all this either. Otherwise the step wouldn't be needed at all.

Reply:
there is still simply no reason to move that step from it's original position to this one

Reply:
You haven't refuted either of my points. 

1. Why have the line there at all when Octoprint won't be connected by default.
2. Why should someone do later versus in the beginning?

Reply:
1. what if you're carrying over from a different firmware like marlin? you may be still connected to your printer?
2. Do I really need to explain to you why things are done in an order and not just randomly? Why aren't you copying the cfg file the home as soon as you download klipper instead of doing it when setup is done? there just isn't a reason to. Even if there's no specifc reason to do it later, there's also none to do it now and you're the one proposing a change without having a reason to do so.